generators/i18n/index.js

```diff
@@ -1,20 +1,38 @@
-const path = require('path'),
-    requiredPath = require('required-path');
+const path = require('path');
+const requiredPath = require('required-path');
+
+const strLang = (file, lang) => `
+if (
+    process.env.BEM_LANG ? process.env.BEM_LANG === '${lang}' :
+    process.env.REACT_APP_BEM_LANG ? process.env.REACT_APP_BEM_LANG === '${lang}' :
+    'en' === '${lang}'
+) {
+    return core()
+        .decl(require('${requiredPath(path.join(file.path, lang))}'))
+        ('${file.cell.entity.id}');
+}
+`;
+
+const langEnvErr = (env, langs) => `process.env.${env} && `
+    + `console.error('No match of process.env.${env} { ' + process.env.${env} + ' } in provided langs: { ${langs.join(', ')} }');`;
+
+const onlyDEV = strings => `
+if (process.env.NODE_ENV === 'development') {
+    ${strings.join('\n')}
+}
+`;
 
 function generateI18n(langs, files) {
-    const strLang = (file, lang) => `
-        if (process.env.BEM_LANG ? process.env.BEM_LANG === '${lang}' : 'en' === '${lang}') {
-            return core()
-                .decl(require('${requiredPath(path.join(file.path, lang))}'))
-                    ('${file.cell.entity.id}');
-        }`;
 
     return files
         .reduce((acc, file) => {
             return acc.concat(langs.map(lang => strLang(file, lang)));
         }, ['(function() {', `var core = require('${requiredPath(path.join(__dirname, "core"))}');`])
         .concat([
-            `console.error('No match of process.env.BEM_LANG {' + process.env.BEM_LANG + '} in provided langs: {${langs}}');`,
+            onlyDEV([
+                langEnvErr('BEM_LANG', langs),
+                langEnvErr('REACT_APP_BEM_LANG', langs)
```

Comment:
So it will warn about BEM_LANG and REACT_APP_BEM_LANG? This is not the same as alias.

Reply:
check `langEnvErr` it has guard, so it will warn you only for one of them.